code/datums/mind.dm

```diff
@@ -63,6 +63,34 @@
 	var/isholy = FALSE //is this person a chaplain or admin role allowed to use bibles
 
 	var/mob/living/enslaved_to //If this mind's master is another mob (i.e. adamantine golems)
+	var/kill_streak = 0 //How many kills we have
+
+/datum/mind/proc/kill_streak_act()
+	var/sound
+	switch(kill_streak)
+		if(3)
+			world << "<B>[name] is on a killing spree!</B>"

```

Comment:
This will probably be useful for me to know later but I probably won't update this PR since it is never going to be merged